SWIPs/swip-23-kademlia.md

```diff
@@ -0,0 +1,238 @@
+---
+SWIP: 23
+title: Swarm node implementer spec - 
+author: Louis Holbrook <dev@holbrook.no> (https://holbrook.no)
+status: Draft
+type: Track Specs
+created: 2019-09-05
+---
+
+## Abstract
+
+This SWIP is a part of a general specification of a Swarm node. The SWIP system is used for convenience only.
+
+## Motivation
+
+Enable other implementations of the swarm node.
+
+## Backwards Compatibility
+
+N/A
+
+## Test Cases
+
+N/A
+
+## Implementation
+
+N/A
+
+## Copyright
+
+Copyright and related rights waived via CC0
+
+## Specification
+
+The Kademlia is both a database (DHT) of known and connected peers on
```

Comment:
I'd put DHT in a definitions table and write it out in full for people who don't know